cranelift/filetests/filetests/isa/s390x/vec-arithmetic.clif

```diff
@@ -1227,10 +1227,11 @@ block0(v0: i64x2, v1: i64x2):
 ;   vlgvg %r3, %v25, 0
 ;   mgrk %r2, %r5, %r3
 ;   lgr %r5, %r2
-;   vlgvg %r2, %v24, 1
-;   vlgvg %r4, %v25, 1
-;   mgrk %r2, %r2, %r4
-;   vlvgp %v24, %r5, %r2
+;   vlgvg %r4, %v24, 1
+;   vlgvg %r2, %v25, 1
+;   mgrk %r2, %r4, %r2
+;   lgr %r4, %r5
+;   vlvgp %v24, %r4, %r2
```

Comment:
I'm curious about the extra move here too.

Reply:
I think removing the copies had the effect of shuffling the order that the registers are picked, which caused the allocator to find a case where it needed to insert a move. Additionally, adding the move on `res_0` back caused the second move to disappear again (even on the RA2-0.7.0 branch), so I'm going to put that one back and call this good.